_team_and_board/5-noffle.md

```diff
@@ -0,0 +1,27 @@
+---
+categories:
+  - team
+layout: team
+title: Stephen Whitmore
+role: "Developer"
+twitter: "https://twitter.com/noffle"
+tags:
+  - core
+current: true
+permalink: /team/noffle
+image: "/assets/noffle.png"
+---
+
+**Stephen Whitmore** works with Digital Democracy to build useful
+open tools that raise the bar of the commons, especially for marginalized
+communities.
+
+Stephen creates and maintains open technology that enables self-determination;
+honours people, not profit; is sustainable for the communities that adopt it;
```

Comment:
reluctantly we need to use the US "honor" not "honour". We can stick to "resources" not "resorces" though, thank goodness that's not a thing.